### ISSUE TITLE
Space between TextField and Left Icon

### ISSUE THREAD

**Issue by CTNCardoso** [CosmicMind/Material#1277]
![Captura de Tela 2019-10-12 às 20 25 41](https://user-images.githubusercontent.com/431215/66708915-13074480-ed2f-11e9-95eb-445dba2ce9aa.png)

Problem is happening with iOS 13.

I tested it on iOS 11 and 12 and it worked perfectly.

**Comment by Shempaiii** [comment]
Any workaround?

**Comment by OrkhanAlikhanov** [comment]
Is that happening after iOS 13?

You can try playing with `textField.textInsets.left = 10`

**Comment by CTNCardoso** [comment]
Yes, it solves the EditText problem, but it generates a problem with label.
<img width="319" alt="Captura de Tela 2019-10-16 às 08 30 43" src="https://user-images.githubusercontent.com/431215/66915378-47108d00-efef-11e9-9ccd-959e779d9959.png">
<img width="322" alt="Captura de Tela 2019-10-16 às 08 30 30" src="https://user-images.githubusercontent.com/431215/66915380-47108d00-efef-11e9-87e7-83f86cd4c9a9.png">

Yes, this issue is only occurring on iOS 13.

**Comment by OrkhanAlikhanov** [comment]
Can you try `textField.leftViewOffset = 32`, default value is 16

**Comment by CTNCardoso** [comment]
Using "textField.textInsets.left = 20" and "textField.leftViewOffset = 32"
<img width="336" alt="Captura de Tela 2019-10-16 às 08 48 09" src="https://user-images.githubusercontent.com/431215/66916579-e171d000-eff1-11e9-8661-f630280b122e.png">

This moved the divider.

**Comment by jbdujardin** [comment]
Is there a fix for this issue?

**Comment by febinrv** [comment]
Yes, the issue is still there 
I have tried 
`usernameTextField.textInsets.left = 8`
`usernameTextField.leftViewOffset = -8`

But not exactly fixed my issue. the divider origin x is wrong now.
